testing/tests/DevExpress.ui.widgets.scheduler/integration.appointments.tests.js

```diff
@@ -2017,13 +1970,12 @@ QUnit.test("Appointment should be returned back if an error occurs during drag (
 
     assert.throws(function() {
         $(this.instance.$element()).find("." + DATE_TABLE_CELL_CLASS).eq(16).trigger(dragEvents.enter);
-        $appointment.trigger(dragEvents.start);
-        $appointment.trigger(dragEvents.end);
+        pointerMock($appointment).start().down().move(10, 10).up();
     }, function(err) {
         var updatedPosition = this.instance.$element().find("." + APPOINTMENT_CLASS).eq(0).position();
 
-        assert.equal(updatedPosition.top, initialPosition.top, "Top is OK");
-        assert.equal(updatedPosition.left, initialPosition.left, "Left is OK");
+        assert.roughEqual(updatedPosition.top, initialPosition.top, 1.1, "Top is OK");
```

Comment:
why `roughEqual`? Does that mean that appointment at coords (1000, 30) can return to coords (1080, 32) and it would be OK?